src/tz_macos.rs

```diff
@@ -119,8 +122,8 @@ mod string_ref {
                 length,
             };
 
+            // SAFETY: `StringRef` is only ever created with a valid `CFStringRef`.
```

Comment:
`clippy::undocumented_unsafe_blocks` wants the SAFETY comment on the block itself, not inside of it.